app/src/main/java/com/example/android/service/BackendClient.java

```diff
@@ -33,4 +38,30 @@ public String fetchAllRestrooms() {
             return null;
         }
     }
+
+    public String fetchUndeployedToiletIds() {
+        final String path = "/undeployedToiletIds";
+        Request request = new Request.Builder().url(_baseUrl + path).build();
+        try (Response response = _client.newCall(request).execute()) {
+            return Objects.requireNonNull(response.body()).string();
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    public String registerToilet(String json) {
+        final String path = "/toilet";
+        RequestBody body = RequestBody.create(json, JSON);
```

Comment:
I am not sure if `RequestBody.create(json, JSON)`  and `RequestBody.create(JSON, json)`, which respectively are your and my usage, will work as the same or not. 
If it also work, we can unify them when merge.